### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#580]
Issue group: Sending
Visibility: 2
Evaluation: Does not support
Success Criteria: 3.3.2: Labels or Instructions
Needs design: Content

### Description
When entering a value for conditional text in a template, a text field input is provided. There is no indication of what type of value is needed to satisfy the condition. It appears that typing “true” will show the conditional text and other text entry will not. At the very least an explanation of what needs to be entered is required. A better approach would be to use a Boolean control (e.g. a checkbox, switch, toggle button, etc.) to indicate if the conditional text should be displayed.

![](https://dl.airtable.com/.attachments/9c1aa7a017ce906a8f518de96e00f011/51665eec/Capturedecranle2022-03-15a13_54_13.png)

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/580?planning-poker) with ZenHub @amazingphilippe @andrewleith @sastels @jzbahrai @jimleroyer

**Comment by amazingphilippe** [comment]
A bit of context for planning poker:

- When we render this page, we only get the name of the placeholder.
- For a11y, we should give a bit more instructions as to what is an acceptable string ("yes", "y", "true", "t", "1", "include", "show", "oui", "vrai", "inclure", "afficher")
- Or we could have the user check a box, and still treat this value as a string. I see no difference, this is just for test emails, or single recipients anyways.
- I any case, we should get a bit more info from notification-utils, that is if the field is a conditional or not. We already have this logic in place, we're just not sending it to admin when we render this form.

**Comment by amazingphilippe** [comment]
Relevant code in utils: 

- This is where we know the type of placeholders: https://github.com/cds-snc/notification-utils/blob/main/notifications_utils/field.py
- This is what we send to our form: https://github.com/cds-snc/notification-utils/blob/main/notifications_utils/template.py

**Comment by sastels** [comment]
has a small bug we should fix
